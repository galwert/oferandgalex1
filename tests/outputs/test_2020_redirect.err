/bin/bash: $'Ð—\031\372wU': command not found
smash error: open failed: Permission denied
/bin/bash: $'\240\237\031\372wU': command not found
smash error: open failed: Permission denied
smash error: open failed: No such file or directory
smash error: open failed: No such file or directory
smash error: open failed: Permission denied
