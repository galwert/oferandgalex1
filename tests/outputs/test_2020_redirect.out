smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash pid is 6402
smash> smash> smash pid is 6402
smash> smash> smash> smash> > mypid
> new
mypid
new
smash> smash> [1] sleep 100& : 6468 0 secs
smash> smash> smash error: kill: invalid arguments
smash> smash> smash error: kill: invalid arguments
smash> smash> > mypid
> new
k
mypid
new
s
t
smash> smash> smash> > mypid
> new
k
mypid
new
p
s
t
smash> smash> smash> before
smash> smash> smash> smash> smash> > mypid
> new
k
mypid
p
s
t
smash> smash> smash> > mypid
> new
k
mypid
new
p
s
t
smash> smash> smash> smash> > mypid
> new
k
mypid
new
p
s
t
smash> smash> smash> smash> smash> smash> smash> smash> smash: got Ctrl-Z
